src/test/ui/consts/const-prop-read-static-in-const.rs

```diff
@@ -1,8 +1,9 @@
+// check-pass
 // compile-flags: -Zunleash-the-miri-inside-of-you
 
 #![allow(dead_code)]
 
-const TEST: u8 = MY_STATIC; //~ ERROR any use of this value will cause an error
```

Comment:
This, too, should get an assertion in main checking the value that was read.
Also Cc @ecstatic-morse as this is a const-prop test; I am not entirely sure what it is supposed to test and why it unleashes Miri so it is unclear how to adjust it.

Reply:
Why am I cc'ed?

Reply:
Didn't you do const-prop stuff? Or am I mixing that up with someone else?

Reply:
Oh I think I meant @wesleywiser, sorry for that!

Reply:
I think this was added just to document the current state when `-Zunleash-the-miri-inside-of-you` is enabled. This change seems fine to me. Asserting in main that the value was read like @RalfJung suggested also sounds like a good idea to me.